src/csm_testing/lib/iuf_classes.py

```diff
@@ -0,0 +1,277 @@
+#
+# MIT License
+#
+# (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+"""
+Defining class for API calls
+"""
+import os
+import base64
+from urllib.error import HTTPError
+import requests
+from kubernetes import client, config  # pylint: disable=import-error
+from keycloak import KeycloakOpenID  # pylint: disable=import-error
+
+
+class AuthException(Exception):
+    """A wrapper for raising an AuthException exception."""
+
+
+class Auth:  # pylint: disable=missing-class-docstring
+    def __init__(self):  # pylint: disable=missing-function-docstring
```

Comment:
I don't think pylint complains when `__init__` methods lack docstrings (I assume because their content would usually be covered in the class docstring itself). We want to avoid adding disable statements for warnings that pylint is not actually raising.

```suggestion
    def __init__(self):
```